_layouts/call.html

```diff
@@ -0,0 +1,22 @@
+<div id="{{ page.priority }}" class="ui {{ page.color | default: 'blue' }} grid aligned segment call">
+    <div class="one wide column"></div>
+
+    {% if page.side == 'left' %}
+      {% include call-description.html %}
+      {% include call-example.html %}
+
+    {% else %} {% comment %} Same but reverse {% endcomment %}
+
+      {% include call-example.html %}
+      {% include call-description.html %}
+
+    {% endif %}
+
+    {% if page.icon == 'compass' %}
```

Comment:
:-1: this is a bad switch. Don't adjust behaviour based on presentational information.